WEB/src/views/ProfileView.vue

```diff
@@ -0,0 +1,92 @@
+<script setup lang="ts">
+  import Profile from '@/components/Profile.vue'
+  import Class from '@/components/Class.vue'
+  import Success from '@/components/Success.vue'
```

Comment:
Vue.js recommends that component names be at least 2 words long to avoid any conflict with native DOM elements. 
ref: https://vuejs.org/style-guide/rules-strongly-recommended.html#base-component-names